TAO/orbsvcs/tests/CosEvent/Basic/Random.cpp

```diff
@@ -344,6 +344,12 @@ RND_Consumer::push (const CORBA::Any &event)
 }
 
 void
+RND_Timer::push (const CORBA::Any &event)
```

Comment:
The question is rather:  if this method isn't ever called, then why is it declared as an override in RND_Timer class at all?

It's a very old and apt heuristic to generate certain class-data into whichever module defines the first "non-inline non-pure-virtual" method,  i.o.w.: the first method that is expected to end up in a .o file.

removing the push from RND_Timer completely would be fine with me, as well.

PS: and keep in mind to never let  operator<<=(unsigned char) in CORBA::Any move to that particular place.